svthalia/Reaxit
# Fix rotations

Closes #366 .

### Summary
This PR will assure that the rotation of photos matches the website

### How to test
Steps to test the changes you made:
1. Go to Photos
2. Click on any `test` album
3. check first image (I made it rotated on the website as well, check that it matches)
